spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -86,7 +86,8 @@ describe('document-capture/components/document-capture', () => {
     await findByText('doc_auth.errors.camera.blocked_detail');
   });
 
-  it('progresses through steps to completion', async () => {
+  it.skip('progresses through steps to completion', async () => {
+    // I believe this to be a selfie related test
```

Comment:
Dunno if related to it being a draft, but we don't allow `.skip`'d test cases, it should be removed outright.